lib/Mojolicious/resources/public/mojo/highlight.js/highlight-mojo-dark.css

```diff
@@ -0,0 +1,90 @@
+﻿.hljs-keyword,
+.hljs-literal,
+.hljs-change,
+.hljs-winutils,
+.hljs-flow,
+.nginx .hljs-title,
+.tex .hljs-special {
+  color:#fcf0a4;
+}
+
+.hljs,
+.hljs-subst {
+  color:#889dbc;
+}
+
+.hljs-string,
+.hljs-title,
+.hljs-type,
+.ini .hljs-title,
+.hljs-tag .hljs-value,
+.css .hljs-rules .hljs-value,
+.hljs-preprocessor,
+.hljs-pragma,
+.ruby .hljs-symbol,
+.ruby .hljs-symbol .hljs-string,
+.ruby .hljs-class .hljs-parent,
```

Comment:
I think we can drop these class names for other languages, per the current version and git history of https://github.com/highlightjs/highlight.js/blob/master/src/styles/dark.css

Reply:
After a closer look, its probably better to recreate this...

Reply:
Updated, it should better highlight some keywords now:

![image](https://user-images.githubusercontent.com/110625/96353982-de795600-1103-11eb-878e-9874c163e725.png)